app/javascript/src/Navigation/components/header/index.jsx

```diff
@@ -0,0 +1,9 @@
+import {AccountSettingsMenu} from 'Navigation/components/header/AccountSettingsMenu'
```

Comment:
It's also possible to export directly:
```suggestion
export {AccountSettingsMenu} from 'Navigation/components/header/AccountSettingsMenu'
```